frontend/src/hooks/useMount.ts

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+/**
+ * Accepts a function that contains imperative, possibly effectful code.
+ * This function will only be run on Mount and Unmount.
+ * @param effect Imperative function that can return a cleanup function.
+ */
+export const useMount = (effect: React.EffectCallback) => {
```

Comment:
Ah just saw the callback and the empty dependency array, I take back my previous question lol

Reply:
This is cleaner though!